ci/cloudbuild/builds/integration.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+source "$(dirname "$0")/../../lib/init.sh"
+source module ci/cloudbuild/builds/lib/bazel.sh
+source module ci/etc/integration-tests-config.sh
+source module ci/lib/io.sh
+
+export CC=clang
+export CXX=clang++
+export CLOUD_SDK_LOCATION=/usr/local/google-cloud-sdk
+readonly EMULATOR_SCRIPT="run_integration_tests_emulator_bazel.sh"
+
+mapfile -t args < <(bazel::common_args)
+bazel test "${args[@]}" --test_tag_filters=-integration-test ...
+
+# Common configuration for all integration tests that follow.
+args+=(
+  "--test_timeout=600"
+  "--test_env=GOOGLE_CLOUD_PROJECT=${GOOGLE_CLOUD_PROJECT}"
+  "--test_env=GOOGLE_CLOUD_CPP_AUTO_RUN_EXAMPLES=yes"
+  "--test_env=GOOGLE_CLOUD_CPP_EXPERIMENTAL_LOG_CONFIG=lastN,100,WARNING"
+  "--test_env=GOOGLE_CLOUD_CPP_ENABLE_TRACING=rpc"
+  "--test_env=CLOUD_STORAGE_ENABLE_TRACING=rpc"
+)
+
+# TODO(#6083): Enable storage integration tests. The difficulty with these is
```

Comment:
nit: for a future PR, maybe enable all the GCS integration tests *except* the ones that use these key files? I think there are only two or three.

Reply:
I was going to do that, but the way the way that `bazel_test_args` is used in [run_integration_tests_emulator_bazel.sh](https://github.com/googleapis/google-cloud-cpp/blob/976a96397d0bb2ef3b8ba22f29acb3e187684a01/google/cloud/storage/ci/run_integration_tests_emulator_bazel.sh#L42) I cannot pass `-//foo/bar/...`. So this script will need to be refactored a bit first, then I'll be able to enable those tests. Just planned to do that in a separate PR